package.json

```diff
@@ -37,16 +36,13 @@
     "koa-socket": "^4.4.0",
     "koa-static": "^4.0.2",
     "minimist": "^1.2.0",
-    "osc-js": "^1.2.2"
+    "nbonjour": "^3.6.1",
```

Comment:
Why are you using `nbonjour` instead of `bonjour`?

Reply:
https://github.com/ardean/spread-the-word is another neat package as well